docs/contributing/pull-requests/branching.md

```diff
@@ -0,0 +1,246 @@
+- Have multiple authors
+- Consist of multiple Pull Requests, each of which have already been reviewed
+- Only consist of merge commits
+
+Since each Pull Request has already been reviewed before being merged into the Long-Lived Feature
+Branch, the feature branch review is more of a sanity check. The reviewer should ensure the
+following:
+
+- The feature branch is ready to be merged into `master`.
+  - Ensure the work has been QA tested, including writing QA Notes in Jira should they be needed, or
+  - Verify the feature is behind a feature flag, and the crossover boundaries go through testing.
+- Review any non-reviewed commits made directly on the branch. These can be either feature work or
+  merge commits. For more details see [Reviewing](#reviewing).
+
+:::warning
+
+Since feature branches do not have the same protections as master, it's technically possible to
+commit directly to the branch or merge a pull request without a up-to-date review. However this
+should be discouraged, and should be avoided whenever possible, the only exception being merge
+commits.
+
+:::
+
+#### Development
+
+To begin development on a feature with a Long-Lived Feature branch, one contributor should create
+the Long-Lived Feature Branch for the piece of functionality and create a **draft** PR from that
+branch into `master`. This name should include the Jira Epic name if applicable.
+
+Each developer should then branch off of that feature branch, creating an "Issue Branch" named with
+the initiating Jira issue (e.g. `sg/SG-123`). We call this an Issue Branch because Jira refers to
+stories and tasks as issues and to differentiate from the Long-Lived Feature Branch above.
+
+:::note
+
+It is important to note that we have decided in this scenario that each of these Issue Branches
+cannot (or will not) be independently tested or released. We have introduced an intermediate Feature
+Branch to collect all of these related changes to allow testing and release as a whole.
+
+:::
+
+As each developer finishes their work, they should open a PR into the Long-Lived Feature Branch. It
+is imperative that every change to the Long-Lived Feature Branch have an approved PR, as otherwise
+all of these individual commits will need to be reviewed prior to the final merge to `master`. The
+developer should tag the appropriate development group to review the PR.
+
+When a developer approves the PR, the PR should be completed, merging that piece of the overall
+change into the Long-Lived Feature Branch.
+
+When all Issue Branches are merged, the `needs-qa` label should be applied to the Long-Lived Feature
+Branch.
+
+#### Syncing the Long-Lived Feature Branch
+
+We typically denote one person as being responsible for keeping the Long-Lived Feature Branch up to
+date.<bitwarden> In most cases this will be the TPC of the pod but can be anyone.</bitwarden> This
+person is responsible for keeping the feature branch reasonably up to date with master, preparing it
+for merge and more.
```

Comment:
We can remove the bitwarden tag since this whole document is now internal.

```suggestion
We typically denote one person as being responsible for keeping the Long-Lived Feature Branch up to
date. In most cases this will be the TPC of the pod but can be anyone. This
person is responsible for keeping the feature branch reasonably up to date with master, preparing it
for merge and more.
```